src/code/audio_seqplayer.c

```diff
@@ -355,7 +385,7 @@ void AudioSeq_SequencePlayerSetupChannels(SequencePlayer* seqPlayer, u16 channel
     SequenceChannel* channel;
     s32 i;
 
-    for (i = 0; i < 0x10; i++) {
+    for (i = 0; i < ARRAY_COUNT(seqPlayer->channels); i++) {
```

Comment:
I'm thinking that since this 16 needs to be the number of bits in a u16 it may be wise to have a define for `NUM_CHANNELS` or smth? (array_count imo usually makes it seem like you can change the array length and things will just work :tm: )